src-js/screens/Admin/screens/Settings/styles/index.scss

```diff
@@ -27,8 +32,16 @@
     // Anime.
     transition: 50ms border-color ease-in-out;
     // Background.
-    background-position: 448px 8px;
+    background-position: 98% 50%;
     background-repeat: no-repeat;
+  
+    &.loading {
+      // Icon.
+      background-image: url('loading.gif');
```

Comment:
do you need to additional file ? is there any native wp component or loading icon from wordlift which you can reuse ?

Reply:
didn't find anything